test/amazonica/test/ecs.clj

```diff
@@ -0,0 +1,7 @@
+(ns amazonica.test.ecs
+  (:require [amazonica.aws.ecs :refer :all]
+            [clojure.set :as set]
+            [clojure.test :refer :all]))
+
+
+
```

Comment:
Do you mind including some basic test functionality here, and copy it into the Examples section of the README for ECS? (Especially since for some reason I could not even find ECS in the official SDK Javadocs.)